### ISSUE TITLE
PhysicsTools.NanoAOD.custom_jme_cff needs to be updated

### ISSUE THREAD

**Issue by makortel** [cms-sw/cmssw#29245]
The PR #29212 broke the workflow 25202.01
```
customising the process with customiseWithTimeMemorySummary from Validation/Performance/TimeMemorySummary
Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/nweek-02620/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_1_X_2020-03-19-1100/bin/slc7_amd64_gcc820/cmsDriver.py", line 56, in &lt;module&gt;
    run()
  File "/cvmfs/cms-ib.cern.ch/nweek-02620/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_1_X_2020-03-19-1100/bin/slc7_amd64_gcc820/cmsDriver.py", line 28, in run
    configBuilder.prepare()
  File "/cvmfs/cms-ib.cern.ch/nweek-02620/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_1_X_2020-03-19-1100/python/Configuration/Applications/ConfigBuilder.py", line 2278, in prepare
    self.pythonCfgCode += self.addCustomiseCmdLine()
  File "/cvmfs/cms-ib.cern.ch/nweek-02620/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_1_X_2020-03-19-1100/python/Configuration/Applications/ConfigBuilder.py", line 903, in addCustomiseCmdLine
    self.executeAndRemember(com)
  File "/cvmfs/cms-ib.cern.ch/nweek-02620/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_1_X_2020-03-19-1100/python/Configuration/Applications/ConfigBuilder.py", line 334, in executeAndRemember
    exec(re.sub(r"([^a-zA-Z_0-9]|^)(process)([^a-zA-Z_0-9])",r"\1self.process\3",command))
  File "&lt;string&gt;", line 1, in &lt;module&gt;
  File "/cvmfs/cms-ib.cern.ch/nweek-02620/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_1_X_2020-03-19-1100/python/PhysicsTools/NanoAOD/custom_jme_cff.py", line 177, in &lt;module&gt;
    jercCHPUF = jetTable.variables.jercCHPUF,
AttributeError: 'PSet' object has no attribute 'jercCHPUF'
```
by removing `jetTable.variables.jercCHPUF`. I presume either it needs to be added back, or `custom_jme_cff` needs to be updated.

**Comment by makortel** [comment]
assign xpog

**Comment by silviodonato** [comment]
solved by  #29257

**Comment by peruzzim** [comment]
+xpog
